problems/variables/problem_nb-no.md

```diff
@@ -0,0 +1,38 @@
+---
+
+# VARIABLER
+
+En variabel er et navn som kan peke til en spesifikk verdi. Variables deklareres ved å bruke `var` etterfulgt av variablens navn.
+
+Her er et eksempel:
+
+```js
+var example;
+```
+
+Variabelen over er **deklarert**, men den er ikke definert (den peker ikke til en spesifikk verdi ennå).
+
+Her er et eksempel som definerer en variabel, ved å peke til en spesifikk verdi:
+
+```js
+var example = 'some string';
+```
+
+# NOTAT
+
+En variabel blir **deklarert** ved bruk av `var` og erlikhetstegn til å **definere** verdien den peker til. Dette kalles som oftes å "sette verdien til en variabel".
+
+## Oppgaven:
+
+Lag en fil som heter `variables.js`.
+
+I den filen skal det deklareres en variabel med navnet `example`.
+
+**Sett verdien `'some string'` inn i variabelen `example`.**
+
+Bruk dermed `console.log()` for å skrive ut verdien til `example` ut til skjermen.
```

Comment:
"Bruk dermed" er litt rart... kanskje bare snu på setningen... "For å skrive ut.... bruk console.log"